kratos/python/add_mesh_to_python.cpp

```diff
@@ -70,6 +60,11 @@ typedef MeshType::NodesContainerType NodesContainerType;
 typedef Geometry<Node<3> >::PointsArrayType NodesArrayType;
 typedef Geometry<Node<3> >::IntegrationPointsArrayType IntegrationPointsArrayType;
 
+array_1d<double,3> GetNormalFromCondition( Condition& dummy )
```

Comment:
the exact implementation of LocalPointWhenProjected is an implementation detail... you can implement the NR correctly basing on the jacobian (which is rectangular) and provides you intrinsically the direction of the normal at any point within the geometry.